morphman/common/centerline_operations.py

```diff
@@ -453,7 +453,8 @@ def get_line_to_change(surface, centerline, region_of_interest, method, region_p
                 break
 
     # Spline the single line
-    line_to_change = compute_splined_centerline(line_to_change, nknots=25, isline=True)
+    nknots = min(line_to_change.GetNumberOfPoints() // 2, 25)
```

Comment:
Godt spørsmål. Nei dette er ikke hentet fra noe sted. Det like så godt vært // 4 her. Det er jo avhengig av resampling size og hvor mye kurvature det er i linjen. Har du noe innsikt fra din master om at vi skal bruke noe et smartere valg?

Reply:
This code is now in `master`, and is important since it is a bug fix. However, feel free to create a small PR with a more robust method for choosing the number of knots.

Reply:
Har ingen gode rule-of-thumbs på det, siden det er så avhengig av faktorer som kurvatur, og er som regel en veldig subjektiv faktor i praksis. 
Formelen jeg disket opp i mastern min var også veldig empirisk, med data fra bare 10 caser. 
Kan godt la denne stå for nå for min del 👍